app/controllers/qa/linked_data_terms_controller.rb

```diff
@@ -14,54 +16,83 @@ def index
-      head :service_unavailable
-      return
-    rescue Qa::ServiceError
-      logger.warn "Internal Server Error - Search query #{query} unsuccessful for#{subauth_warn_msg} authority #{vocab_param}"
-      head :internal_server_error
-      return
-    rescue RDF::FormatError
-      logger.warn "RDF Format Error - Results from search query #{query} for#{subauth_warn_msg} authority #{vocab_param} " \
-                  "was not identified as a valid RDF format.  You may need to include the linkeddata gem."
-      head :internal_server_error
-      return
-    end
+  def search
+    terms = @authority.search(query, subauth: subauthority, language: language, replacements: replacement_params)
     cors_allow_origin_header(response)
     render json: terms
+  rescue Qa::ServiceUnavailable
+    logger.warn "Service Unavailable - Search query #{query} unsuccessful for#{subauth_warn_msg} authority #{vocab_param}"
+    head :service_unavailable
+  rescue Qa::ServiceError
+    logger.warn "Internal Server Error - Search query #{query} unsuccessful for#{subauth_warn_msg} authority #{vocab_param}"
+    head :internal_server_error
+  rescue RDF::FormatError
+    logger.warn "RDF Format Error - Results from search query #{query} for#{subauth_warn_msg} authority #{vocab_param} " \
+                "was not identified as a valid RDF format.  You may need to include the linkeddata gem."
+    head :internal_server_error
   end
 
-  # Return all the information for a given term
+  # Return all the information for a given term given an id or URI
+  # get "/show/linked_data/:vocab/:id"
+  # get "/show/linked_data/:vocab/:subauthority/:id
   # @see Qa::Authorities::LinkedData::FindTerm#find
   def show # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
-    begin
-      term = @authority.find(id, subauth: subauthority, language: language, replacements: replacement_params)
-    rescue Qa::TermNotFound
-      logger.warn "Term Not Found - Fetch term #{id} unsuccessful for#{subauth_warn_msg} authority #{vocab_param}"
-      head :not_found
-      return
-    rescue Qa::ServiceUnavailable
-      logger.warn "Service Unavailable - Fetch term #{id} unsuccessful for#{subauth_warn_msg} authority #{vocab_param}"
-      head :service_unavailable
-      return
-    rescue Qa::ServiceError
-      logger.warn "Internal Server Error - Fetch term #{id} unsuccessful for#{subauth_warn_msg} authority #{vocab_param}"
-      head :internal_server_error
-      return
-    rescue RDF::FormatError
-      logger.warn "RDF Format Error - Results from fetch term #{id} for#{subauth_warn_msg} authority #{vocab_param} " \
-                  "was not identified as a valid RDF format.  You may need to include the linkeddata gem."
-      head :internal_server_error
-      return
-    end
+    term = @authority.find(id, subauth: subauthority, language: language, replacements: replacement_params)
```

Comment:
Changes to this method are not functional changes.  The method operates exactly the same.  The refactor makes the code cleaner and easier to read.